src/System.Private.Uri/src/System.Private.Uri.builds

```diff
@@ -9,17 +9,18 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Private.Uri.csproj">
-      <OSGroup>Windows_NT</OSGroup>
```

Comment:
the latter, this is just an artifact of multiple edits. I can move it back if you want me to.